src/forms/radio/RadioButton.tsx

```diff
@@ -8,21 +8,24 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import * as React from 'react';
-import * as utils from '../../utils';
+import * as React from "react";
+import * as utils from "../../utils";
 import {ReactNode} from "react";
```

Comment:
Everything from 'react' has already imported above. you can use ReactNode as React.ReactNode instant of importing it here again.

Reply:
I actually referred following RadioButtonGroup code.
https://github.com/EMCECS/clarity-react/blob/master/src/forms/radio/RadioButtonGroup.tsx#L14

It has both the lines

Reply:
But we can still change imports here if it is more appropriate.